gem/view_gem_dag.py

```diff
@@ -0,0 +1,86 @@
+from enum import Enum
+from graphviz import Digraph
+from re import search
+
+class GraphType(Enum):
+    """Type of graph to create
+    PARENT_CHILD: Graph is composed of parent-child relationships only and each node is labelled with all information about the node (its type and all its slots including 'children').
```

Comment:
If I remember correctly, the correct Sphinx syntax for giving docstring for non-function members is a string in the line immediately following the assignment, but please look it up!

Reply:
Please break your lines.

Yes, we disable the flake8 rule which would force every line to be shorter than 80 characters, because sometimes that would make code really ugly, but for the most part we should keep the 80-character limit.